apis/v1alpha1/route_types.go

```diff
@@ -55,6 +55,12 @@ type RouteGateways struct {
 	GatewayRefs []GatewayReference `json:"gatewayRefs,omitempty"`
 }
 
+// Port defines a network port.
+//
+// +kubebuilder:validation:Minimum=1
+// +kubebuilder:validation:Maximum=65535
+type Port uint32
```

Comment:
Suggest that we call this type `PortNumber` for clarity.

Reply:
`PortNumber` feels a little more verbose than it needs to be. What is the reasoning behind the suggestion?

Reply:
In k/k our [Port types include a ton of fields](https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L3702), including a name and protocol. I think using `PortNumber` here may make the simplicity of this type more clear.

Reply:
This is a type alias and not even a struct. Do we still need to be explicit even then?

Reply:
It's a small detail, I'd just imagine that anyone browsing through the types might see `Port` in a `forwardTo` and wonder if that included all the things that `ServicePort` did or if it was just a port number. Obviously it's not difficult to figure that out, just takes an extra step.

Reply:
I see. I've updated this to be `PortNumber`. Like you said, it is a small detail and I don't feel too strongly either ways.

Reply:
Yeh, it's clarity ... `PortNumber` is immediately obvious and evocative of its meaning.